package.json

```diff
@@ -64,6 +64,7 @@
     "@galoymoney/react-native-geetest-module": "^0.1.3",
     "@react-native-async-storage/async-storage": "^1.19.3",
     "@react-native-clipboard/clipboard": "^1.12.1",
+    "@react-native-community/geolocation": "^3.1.0",
```

Comment:
does this have any effect on the permission required by the App stores?

Reply:
Android is already dealt with previously with this in the manifest
`<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />`

iOS i'm 99% sure. The docs say this is required and we already have it
`NSLocationWhenInUseUsageDescription`
This is not needed cause our deployment target is > 10
`NSLocationAlwaysUsageDescription`
And this one sucks and I really don't want to add it. The docs say it's mandatory, but I've tested it on iOS and it worked regardless
`NSLocationAlwaysAndWhenInUseUsageDescription`